src/runtime/config/configuration-qemu.toml.in

```diff
@@ -389,6 +389,9 @@ valid_entropy_sources = @DEFVALIDENTROPYSOURCES@
 # be default_memory.
 #enable_guest_swap = true
 
+# use legacy serial for guest console if available and implemented for architecture. Default false
+#use_legacy_serial = true
```

Comment:
Maybe document it in [docs/how-to/how-to-set-sandbox-config-kata.md](https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-set-sandbox-config-kata.md) as well ?

Reply:
right, I'll look into it, thanks!

Reply:
> right, I'll look into it, thanks!

I see you've added a commit for the annotation which looks good to me. Thanks !